include/nbl/builtin/hlsl/central_limit_blur/box_blur.hlsl

```diff
@@ -0,0 +1,297 @@
+#ifndef _NBL_BUILTIN_HLSL_CENTRAL_LIMIT_BLUR_BOX_BLUR_INCLUDED_
+#define _NBL_BUILTIN_HLSL_CENTRAL_LIMIT_BLUR_BOX_BLUR_INCLUDED_
+
+#include <nbl/builtin/hlsl/central_limit_blur/common.hlsl>
+
+#include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
+#include <nbl/builtin/hlsl/workgroup/arithmetic.hlsl>
+#include <nbl/builtin/hlsl/workgroup/scratch_size.hlsl>
+
+
+// https://github.com/microsoft/DirectXShaderCompiler/issues/6144
+static nbl::hlsl::uint32_t3 /*nbl::hlsl::glsl::*/gl_WorkGroupSize() { return uint32_t3( WORKGROUP_SIZE, 1, 1 ); }
```

Comment:
we even fwd declare this function in some header I think, maybe the `glsl_compat` stuff?